### DISCUSSION TITLE
iseq does not work with integers from script

### DISCUSSION THREAD

**Discussion by petrasl1976** [ipxe/ipxe#330] [General]
Hi,

If i compare integer value from shell, it works perfectly:
```
set var1:int8 0
iseq ${var1} 1 && echo yes || echo no
no
inc var1
iseq ${var1} 1 && echo yes || echo no
yes
```
But if i put it in the boot script i get this:
```
Usage
  iseq <value1> <value2>
```
I have tried to to use `var2:int8` instead of `1` in the `iseq` command, also tried to "convert to string" like this ${var1:uristring} - result the same.

I use the latest version of iPXE

Regards
PetrasL

**Comment by mcb30** [comment]
Try posting a full copy of your iPXE script, so that someone can find the problem in your script.  (Ideally as an attachment, to avoid any possible copy-paste errors.)
